docs/source/v1/Cryosphere/index.rst

```diff
@@ -0,0 +1,45 @@
+E3SMv1 (Cryosphere)
+====================
+
+The `E3SM version 1 cryosphere simulation campaign <https://e3sm.org/research/cryosphere-ocean/v1-cryosphere-ocean/>`_ includes standard set of 
+Coupled Model Intercomparison Project Phase 6 (`CMIP6 <https://www.geosci-model-dev.net/9/1937/2016/>`_) 
+Diagnosis, Evaluation, and Characterization of Klima (DECK) simulations. 
+These E3SM simulations at standard resolution have been completed and the data is publicly available.
+
+Data Resolution:
+
+The standard resolution configuration of E3SM v1.0 water cycle configuration has approximate horizontal resolution of 
+1 deg latitude by 1 deg longitude in atmosphere (110 km grid spacing), 
+with ocean and sea ice grid of 60 km in the mid-latitudes and 30 km at the equator and poles, 
+and river transport at 55 km horizontal resolution. 
+This model configuration is described in  
+`“v1 1 deg CMIP” <https://e3sm.org/model/scientifically-validated-configurations/v1-configurations/v1-1-deg-cmip6/>`_ page 
+in `Scientifically Validated Configurations <https://e3sm.org/model/scientifically-validated-configurations/>`_.
+
+For more details, 
+refer to `Coupled E3SM v1 Model Overview <https://e3sm.org/overview-paper-on-coupled-model/>`_ or to the reference papers:
+
+* *The DOE E3SM Coupled Model Version 1: Overview and Evaluation at Standard Resolution* `doi: 10.1029/2018MS001603 <https://doi.org/10.1029/2018MS001603>`_
+* *Description of historical and future projection simulations by the global coupled E3SMv1.0 model as used in CMIP6* `doi:10.5194/gmd-15-3941-2022 <https://doi.org/10.5194/gmd-15-3941-2022>`_
+* *Ensemble Spread Behavior in Coupled Climate Models: Insights From the Energy Exascale Earth System Model Version 1 Large Ensemble* `doi:10.1029/2023MS003653 <https://doi.org/10.1029/2023MS003653>`_
+* *The DOE E3SM Coupled Model Version 1: Description and Results at High Resolution* `doi:10.1029/2019MS001870 <https://doi.org/10.1029/2019MS001870>`_
+
+Experiments:
+
+The datasets include the following experiments.
+
+* 1850 PI Control
+* 1850 PI control with ice shelf melt fluxes
+* CORE-IAF control
+* CORE-IAF with ice shelf melt fluxes 
+* 1850 PI control with ice shelf melt fluxes + 3DGM
+* CORE-IAF with ice shelf melt fluxes + 3DGM
+* 1850 PI control with ice shelf melt fluxes, bug fix run
+* 1850 PI control with ice shelf melt fluxes + 3DGM, bug fix run
```

Comment:
Should these have any additional explanation? I suppose we could use the full titles of the Confluence pages linked on https://e3sm.atlassian.net/wiki/spaces/ED/pages/4495441922/V1+Simulation+backfill+WIP, like "Low-Res, Pre-Industrial Control Run (60to30km B-cases) - A_WCYCL1850-DIB-ISMF; snowcapping"